libgalois/include/katana/analytics/Utils.h

```diff
@@ -13,57 +13,25 @@ namespace katana::analytics {
 //! Used to pick random non-zero degree starting points for search algorithms
 //! This code has been copied from GAP benchmark suite
 //! (https://github.com/sbeamer/gapbs/blob/master/src/benchmark.h)
-template <typename Graph>
-class SourcePicker {
-  static const uint32_t kRandSeed;
+class KATANA_EXPORT SourcePicker {
```

Comment:
Not related to this PR and no need to fix now, but this is a good a time as any to give my yearly "`utils` is a horrible name" speech. Utility files/classes don't have a clear focus so they tend to just accumulate unrelated functionality and become a site of future circular dependencies.

Now, I'm off my soapbox.

Some of these functions might make more sense being declared closer to PropertyFileGraph. (And SourcePicker seems undergeneralized since the seed is just fixed.)

Reply:
I fixed the SourcePicker issue and added a TODO for the "utils" issue.